### PR TITLE
update for v5.0 release

### PR DESCRIPTION
**Pull request by janjust**

### CODE REVIEW

**Comment by janjust** [comment]
note: we still have to edit the major changes.

File: nightly/index.php [hunk 1]
```diff
@@ -16,14 +16,19 @@
...
 <p> <strong>Prior stable release series</strong>
 
+<li><a href="v4.1.x/"><strong>v4.1.x series</strong></a>: These are
+images from the prior stable v4.1.x series.
+<strong><font color="red">This code base is still
+supported.</font></strong></li>
+
 <li><a href="v4.0.x/"><strong>v4.0.x series</strong></a>: These are
 images from the prior stable v4.0.x series.
 <strong><font color="red">This
```
Comment (jsquyres) [inline nightly/index.php:34]: Do you need to change the "current development" section (below) to remove the v5.0.x series?

File: nightly/index.php [hunk 2]
```diff
@@ -16,14 +16,19 @@
...
 
 <p> <strong>Prior stable release series</strong>
 
+<li><a href="v4.1.x/"><strong>v4.1.x series</strong></a>: These are
+images from the prior stable v4.1.x series.
+<strong><font color="red">This code base is still
+supported.</font></strong></li>
+
 <li><a href="v4.0.x/"><strong>v4.0.x series</strong></a>: These are
 images from the prior stable v4.0.x series.
```
Comment (jsquyres) [inline nightly/index.php:33]: Let's move v4.0.x into the "Older" sections (both in the main text content and in the left-hand nav), and just have v4.1.x in the "Prior stable" category.

File: software/ompi/v5.0/timeline-graph.php [hunk 3]
```diff
@@ -28,22 +28,23 @@
...
+// milestone("v5.0.0rc7", "2022-05-13", $data, $vpos);
+// milestone("v5.0.0rc8", "2022-09-29", $data, $vpos);
+// milestone("v5.0.0rc9", "2022-10-21", $data, $vpos);
+// milestone("v5.0.0rc10", "2023-02-02", $data, $vpos);
+// milestone("v5.0.0rc11", "2023-04-06", $data, $vpos);
+// milestone("v5.0.0rc12", "2023-05-19", $data, $vpos);
+// milestone("v5.0.0rc13", "2023-09-29", $data, $vpos);
+// milestone("v5.0.0rc14", "2023-10-17", $data, $vpos);
+// milestone("v5.0.0rc15", "2023-10-19", $data, $vpos);
+// milestone("v5.0.0rc16", "2023-10-24", $data, $vpos);
```
Comment (jsquyres) [inline software/ompi/v5.0/timeline-graph.php:46]: Don't comment these out; delete them.  There's no point in keeping them; they're in git history if we need them (but we won't).

File: software/ompi/v5.0/version.inc [hunk 4]
```diff
@@ -11,11 +11,13 @@ $download_prefix="https://download.open-mpi.org/release/open-mpi/v" . $release_s
...
 /* releases must be ordered newest to oldest */
-$releases = array();
+$releases = array("5.0.0");
+
 /* prereleases must be ordered newest to oldest.  All prereleases
    will be shown, so make an empty array when the official release
-   is added to releases above */
-$prereleases = array("5.0.0rc16", "5.0.0rc15", "5.0.0rc14", "5.0.0rc13", "5.0.0rc12", "5.0.0rc11", "5.0.0rc10", "5.0.0rc9", "5.0.0rc8", "5.0.0rc7", "5.0.0rc6", "5.0.0rc5", "5.0.0rc4", "5.0.0rc3", "5.0.0rc2", "5.0.0rc1");
+   is added to releases above
+   $prereleases = array("5.0.0rc16", "5.0.0rc15", "5.0.0rc14", "5.0.0rc13", "5.0.0rc12", "5.0.0rc11", "5.0.0rc10", "5.0.0rc9", "5.0.0rc8", "5.0.0rc7", "5.0.0rc6", "5.0.0rc5", "5.0.0rc4", "5.0.0rc3", "5.0.0rc2", "5.0.0rc1"); */
```
Comment (jsquyres) [inline software/ompi/v5.0/version.inc:19]: Delete, don't comment out.

**Review by jsquyres — CHANGES_REQUESTED** [review]

File: nightly/index.php (hunk 2 above)
Comment (janjust) [reply nightly/index.php:33]: So we're officially not in bugfix on v4.0? I can remove that text?

Comment (hppritcha) [reply nightly/index.php:33]: i'd vote for moving v4.0.x out of still supported and into older versions

Comment (janjust) [reply nightly/index.php:33]: Sorry I deleted my comment asking about the text bug-fix-only mode. It said that v4.0 is still supported but in bug fix only mode. Unless we really want to not support it anymore, I think it should remain here.

Comment (janjust) [reply nightly/index.php:33]: Alternatively, I can move it to older version, but leave the text

Comment (jsquyres) [reply nightly/index.php:33]: We discussed this in Slack: perhaps I was too aggressive here.  Let's leave it in the "prior stable" section for now.  We can always file another PR later if we want to move 4.0 out of this section.

File: software/ompi/v5.0/timeline-graph.php (hunk 3 above)
Comment (janjust) [reply software/ompi/v5.0/timeline-graph.php]: done

File: software/ompi/v5.0/version.inc (hunk 4 above)
Comment (janjust) [reply software/ompi/v5.0/version.inc]: done

File: nightly/index.php (hunk 1 above)
Comment (janjust) [reply nightly/index.php:34]: done

**Comment by janjust** [comment]
We still have the major changes section to update, and I'll squash commits when ready

**Review by jsquyres — APPROVED** [review]
Go ahead and squash.

**Comment by janjust** [comment]
@jsquyres @awlauria I changed the major-changes section for v5.0.0, considering we have a lot of smaller changes I decided to leave those out and only point out a few notable ones. If you think I need to add more, please point them out and I'll add them.

File: software/ompi/major-changes.php [hunk 5]
```diff
@@ -9,6 +9,57 @@
 present an incompatibility vs. a prior version of Open MPI, or cause
 some other type of user-noticeable change in behavior.</em>
 
+<h2>v5.0.x series</h2>
```
Comment (jsquyres) [inline software/ompi/major-changes.php:12]: Why do we still have this page?  Future action item: this page should move into the RST docs.

File: software/ompi/major-changes.php [hunk 6]
```diff
@@ -9,6 +9,57 @@
...
+  <li> CUDA support in <em>OFI</em> MTL. </li>
+  <li> Added a new <em>Accelerator</em> framework. <br> CUDA-specific code
+      was replaced with a generic framework that standardizes various
+      device features such as copies or pointer type detection. </li>
+  <li> HAN collectives enabled by default </li>
+</ul>
+
+<p></p>
+
+<li> Open MPI v5.0.x is ABI compatible with Open MPI 4.1.x.</li>
```
Comment (jsquyres) [inline software/ompi/major-changes.php:55]: and 4.0.x.

File: software/ompi/major-changes.php [hunk 7]
```diff
@@ -9,6 +9,57 @@
...
+
+</ol></p>
+
+For the full list of features and updates please refer to the v5.0.x news section.
```
Comment (jsquyres) [inline software/ompi/major-changes.php:59]: You should probably link to this.

**Review by awlauria — APPROVED** [review]

File: software/ompi/major-changes.php (hunk 5 above)
Comment (janjust) [reply software/ompi/major-changes.php:12]: So should I just ommit everything here, and just link to v5.0.x news?

Comment (jsquyres) [reply software/ompi/major-changes.php:12]: It's probably ok to leave this for now; let's get 5.0.0 out the door.  We can clean up this (i.e., delete/redirect this page to an appropriate page in the RST docs) after the fact.  Sorry -- that's what I was trying to say as "Future action item: ...".

File: software/ompi/major-changes.php (hunk 7 above)
Comment (janjust) [reply software/ompi/major-changes.php:59]: Similar to the previous comment, why don't I remove all the features, and link to v5.0 docs, and only leave building/linking, prrte, and ABI compatilibity changes.

Comment (jsquyres) [reply software/ompi/major-changes.php]: I think there's 3 options

1. Keep what is here, fix the comments I made.
2. Do the entire cleanup (i.e., put in the redirect for `major-changes.php` to somewhere in the docs),
3. Keep the `major-changes.php` page, but link the v5.0 stuff to the appropriate places in the RST docs.

You pick which one you want to do right now.  😄

**Review by jsquyres — APPROVED** [review]
Please squash before merging.

**Comment by janjust** [comment]
done

**Pull request merged**